draft-ietf-grow-peering-api.md

```diff
@@ -683,9 +736,8 @@ The authors would like to thank their collaborators, who implemented API version
 * David Tuber (Cloudflare)
```

Comment:
Feel free to remove Fastly as affiliation

Reply:
Done